net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch

```diff
@@ -0,0 +1,53 @@
+Fix compatibility with kwayland-5.93.0
+
+An include path that was previously being relied on isn't present anymore with the new release
```

Comment:
Please include a link to the Gentoo bug and upstream bug/PR at the top of the patch (do this for all patches in future too)

Reply:
Fixed